src/workloads/scale/LLTInsert.yml

```diff
@@ -0,0 +1,248 @@
+      Type: CrudActor
+      Database: *dbname
+      ClientName: Update
+      Threads: 1
+      Phases:
+        OnlyActiveInPhases:
+          Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 3}}]
+          NopInPhasesUpTo: 10
+          PhaseConfig:
+            ExternalPhaseConfig:
+              Path: ../../phases/scale/LLTPhases.yml
+              Key: InsertPhase.50
+
+  - TemplateName: InsertTemplate.800.50
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "InsertBaseline.100"}}
+      Type: CrudActor
+      Database: *dbname
+      ClientName: Update
+      Threads: 1
+      Phases:
+        OnlyActiveInPhases:
+          Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 5}}]
+          GlobalRate: 3200 per second
+          NopInPhasesUpTo: 10
+          PhaseConfig:
+            ExternalPhaseConfig:
+              Path: ../../phases/scale/LLTPhases.yml
+              Key: InsertPhase.50
+
+  # Rotate the mongo logs.
+  - TemplateName: LogRotateTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "LogRotate"}}
+      Type: AdminCommand
+      Threads: 1
+      Phases:
+        OnlyActiveInPhases:
+          Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1}}]
+          NopInPhasesUpTo: 10
+          PhaseConfig:
+            Repeat: 1
+            Operations:
+              - OperationName: AdminCommand
+                OperationIsQuiet: true
+                OperationCommand:
+                  logRotate: 1
+
+  # SetProfiling level to 0, -1, this will log all operations.
+  # Set back to 0, 100 for default.
+  - TemplateName: SetProfilingLevelTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "EnableProfiling"}}
+      Type: RunCommand
+      Database: test
+      Threads: 1
+      Phases:
+        OnlyActiveInPhases:
+          Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1}}]
+          NopInPhasesUpTo: 10
+          PhaseConfig:
+            Repeat: 1
+            Operations:
+              - OperationName: RunCommand
+                OperationIsQuiet: true
+                OperationCommand:
+                  profile: {^Parameter: {Name: "Name", Default: 0}}
+                  slowms: {^Parameter: {Name: "Name", Default: -1}}
+
+Actors:
+#  # Rotate logs and set the profiling level for debugging.
+#  - ActorFromTemplate:
+#      TemplateName: LogRotateTemplate
+#      TemplateParameters: {}
+#
+#  - ActorFromTemplate:
+#      TemplateName: SetProfilingLevelTemplate
+#      TemplateParameters: {}
+
+  - ActorFromTemplate:
+      TemplateName: LoaderTemplate
+      TemplateParameters:
+        Name: InitialLoad
+
+  - ActorFromTemplate:
+      TemplateName: LoaderTemplate
+      TemplateParameters:
+        Name: InitialLoadWithIndexes
+        DocumentCount: *SecondaryNumDocs
+        OnlyActiveInPhase: 2
+        Indexes: {}
+
+  # <-----  4 threads @ 100 ops / second in batches of 50
```

Comment:
Where do we set 100 ops / second? We don't have global rate in InsertTemplate.100.50.